packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -138,26 +125,23 @@ function pickEntity(viewer, e) {
 
 const scratchStopTime = new JulianDate();
 
-function trackDataSourceClock(timeline, clock, dataSource) {
+function trackDataSourceClock(timeline, dataSource) {
```

Comment:
Optional change: It might be nice to rename this function now to something like `linkTimelineToDataSourceClock`? it's a little more verbose but describes better the intent?

Reply:
ok